src/lib/openapi/spec/environment-project-schema.ts

```diff
@@ -8,22 +8,37 @@ export const environmentProjectSchema = {
     properties: {
         name: {
             type: 'string',
+            example: 'development',
+            description: 'The name of the environment',
         },
         type: {
             type: 'string',
+            example: 'production',
+            description:
+                'The type of environment. Typically either development (non-production) or production',
         },
         enabled: {
             type: 'boolean',
+            example: true,
+            description:
+                '`true` if the environment is enabled for the project, otherwise `false`',
         },
         protected: {
             type: 'boolean',
+            example: false,
+            description:
+                '`true` if the environment is not removable, otherwise `false`',
         },
         sortOrder: {
             type: 'number',
+            example: 1,
+            description:
+                'Priority of the environment in a list of environments, the lower value, the higher up in the list the environment will appear',
```

Comment:
I think I've asked this before, but there's no minimum or maximum for this right? Negative numbers are also okay? Do they have to be integers?

```suggestion
            example: 1,
            description:
                'Priority of the environment in a list of environments, the lower the value, the higher up in the list the environment will appear',
```

Reply:
No min or max, should be integers yes, I don't think we accept decimals.

Reply:
Nice, thanks! In that case, I think there's only a missing "the" in the sentence (as mentioned in the suggestion) 😄